platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HttpRequestUrl.java

```diff
@@ -17,14 +17,19 @@ private HttpRequestUrl() {
    * @param querySize   the query size of the resource request
    * @return the adapted resource url
    */
-  public static String buildResourceUrl(@NonNull String host, String resourceUrl, int querySize) {
+  public static String buildResourceUrl(@NonNull String host, String resourceUrl, int querySize, boolean offline) {
     if (isValidMapboxEndpoint(host)) {
       if (querySize == 0) {
         resourceUrl = resourceUrl + "?";
       } else {
         resourceUrl = resourceUrl + "&";
       }
-      resourceUrl = resourceUrl + "sku=" + Mapbox.getSkuToken();
+
+      if (offline) {
+        resourceUrl = resourceUrl + "offline=true";
```

Comment:
I think we still want to send the sku token with offline requests for now, but I'm going to defer to @zugaldia on that.

Reply:
This wasn't completely clear to me when reading the requirements. Any idea @zugaldia?

Reply:
Correct. We still want to attach the SKU token.

Reply:
Code and tests have been updated to reflect above